src/schemas/settings/v2/index.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod'
+import {
+  AppearanceSchema,
+  ProxySettingsSchema,
+  RecorderSettingsSchema,
+  UsageReportSettingsSchema,
+  WindowStateSchema,
+} from '../v1'
+
+export {
+  AppearanceSchema,
+  ProxySettingsSchema,
+  RecorderSettingsSchema,
+  UsageReportSettingsSchema,
+  WindowStateSchema,
+}
+
+export const AppSettingsSchema = z.object({
+  version: z.literal('2.0'),
```

Comment:
This is the same as v1, but I left some breadcrumbs to help us with the first migration when the time comes so no functional change at this time.